test/src/api/responder03Test.php

```diff
@@ -3549,4 +3549,112 @@ public function test_interface_game_055() {
         $expData['gameActionLogCount'] += 3;
         $retval = $this->verify_api_loadGameData($expData, $gameId, 10);
     }
+
+    /**
+     * @depends responder00Test::test_request_savePlayerInfo
+     * @group fulltest_deps
```

Comment:
Oh, no rush, but we should take this line out before pushing; that was my mistake.